pkg/apis/eksctl.io/v1alpha5/assets/schema.json

```diff
@@ -56,8 +56,8 @@
         },
         "permissionsBoundary": {
           "type": "string",
-          "description": "ARN of the permissions boundary to associate",
-          "x-intellij-html-description": "ARN of the permissions boundary to associate"
+          "description": "ARN of the permissions' boundary to associate",
```

Comment:
The comment for this was updated, but the schema was not regenerated. :/ And that was me in the lint pr. :D

Reply:
I'll allow' it